lib/team.rb

```diff
@@ -0,0 +1,12 @@
+class Team
+  attr_reader :team_id, :franchise_id, :team_name, :abbreviation, :stadium, :team_link
+
+  def initialize(team_params)
+    @team_id = team_params[:team_id].to_i
+    @franchise_id = team_params[:franchiseId].to_i
```

Comment:
we should discuss with teacher good convention on changing to integer at the instance variable